crates/subspace-core-primitives/src/lib.rs

```diff
@@ -430,6 +430,8 @@ pub enum RootBlock {
         segment_index: u64,
         /// Merkle root of the records in a segment.
         records_root: Sha256Hash,
+        /// Merkle root of object mappings root
+        object_mappings_root: Sha256Hash,
```

Comment:
You can't just randomly add fields here, you'll break everything. This enum is versioned exactly to distinguish between different variants and methods are present to abstract away differences between them.

Reply:
If we go with adding it to this data structure then yes